zebrad/tests/common/configs/v1.3.1.toml

```diff
@@ -0,0 +1,73 @@
+# Default configuration for zebrad.
```

Comment:
Nitpick: the next release is 1.4.0: #7764

Reply:
Updated in https://github.com/ZcashFoundation/zebra/pull/7844/commits/8ef372dd22edddf66b9503e81b9e576b628ef657.